wrappers/python/nntile/model/gpt2.py

```diff
@@ -252,6 +263,17 @@ def __init__(
                 vocab_embed_dim_tile,
                 next_tag,
             )
+        elif self.dtype == "fp32_fast_fp16":
+            wpe_layer, next_tag = Embedding.generate_simple(
+                positional_ids.value,
+                Tensor_fp32_fast_fp16,
```

Comment:
all of these long if-elif statements can be simplified: add a dict to map string like 'fp32_fast_tf32' to a proper `Tensor_*` type and just do `map[dtype]` to get a proper tensor type.